push_data_to_ccos/get_community_team_data.py

```diff
@@ -50,15 +55,15 @@ def generate_databag():
         ]
     }
     """
-    print("Pulling from Asana and generating databag...")
+    logger.log(logging.INFO, "Pulling from Asana and generating databag...")
     databag = {"projects": [], "community_builders": []}
 
     members = ASANA_CLIENT.tasks.find_by_section(
         ASANA_PROJECT_GID, opt_fields=["name", "custom_fields"]
     )
-    print("    Team members pulled.")
+    logger.log(logging.INFO, "    Team members pulled.")
+    logger.log(logging.INFO, "    Processing team members...")
```

Comment:
The main advantage of using the customised logger is to not have to indent log messages manually. The logger will take care of indentation automatically.

Reply:
Yes, I will do that immediately. Also, the logger is really good it almost covers everything.